src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final String[] COLORS = {"red", "pink", "yellow", "black", "green", "blue"};
+
+    public String getRandomColor() {
+        Random random = new Random();
```

Comment:
The `Random` object should be a private class-level field instead of being instantiated in the method. This avoids creating a new `Random` instance every time the method is called, which is unnecessary and less efficient.